app/client/src/components/accordion.tsx

```diff
@@ -3,6 +3,12 @@ import { useCallback, useState } from 'react';
 // types
 import type { ReactNode } from 'react';
 
+/*
+## Components
+*/
+
+export default Accordion;
+
 export function Accordion({ children }: { children: ReactNode }) {
```

Comment:
```suggestion
export default function Accordion({ children }: { children: ReactNode }) {
```

Why not just put the default in when you export the function component (similarly for all other components in this PR)?

Reply:
This is just a pattern I was testing out, so that the default component can be either imported as default or along with other named exports, ie.
`import Accordion, { AccordionItem } from 'components/accordion';`
and
`import { Accordion, AccordionItem } from 'components/accordion';`
would both work.

I'm open to either pattern.

Reply:
Oh ok. I think we can remove the `export default Accordion`. I could see doing that if we were developing an open source API just to make it easy for a large number of developers. 

I think in our case we should either do the `export default Accordion(...)` or not use default exports. 

I looked at clean school bus (developed by Courtney) and he is not using default exports at all. So maybe we should just switch over to not using default anymore.

Reply:
Sounds good, I'll just remove default exports.